frontend/src/component/layouts/TabbedLayout.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component, ReactChild } from 'react';
```

Comment:
I think tab layout is more like this: https://material.angular.io/components/tabs/overview
This component has drawer. I am not sure whether it qualifies for tab component. I recommend directly put them inside admin page.

What are your thoughts on this?

Reply:
To clarify, I recommend closing this PR and create an empty admin page PR instead.

Reply:
Okay, got it!